src/pudl/glue/ferc1_eia.py

```diff
@@ -253,12 +253,25 @@ def get_plants_ferc1_raw(
         plant_dfs.append(plant_df)
 
     all_plants = pd.concat(plant_dfs)
+    # add the utility_name_ferc1
+    util_map = get_utility_map_pudl()
+    unique_utils_ferc1 = util_map.loc[
+        util_map.utility_id_ferc1.notnull(), ["utility_id_ferc1", "utility_name_ferc1"]
+    ].drop_duplicates(subset=["utility_id_ferc1"])
+    all_plants = all_plants.merge(
+        unique_utils_ferc1,
+        on=["utility_id_ferc1"],
+        how="left",
+        validate="m:1",
+    )
+    # grab the most recent plant record
```

Comment:
this was meant to go into #1988... it simply adds the utility id name to the csv that is generated with missing plants